src/hooks/useSnackbar/useSnackbar.stories.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Zextras <https://www.zextras.com>
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+import React from 'react';
+
+import { Meta, StoryObj } from '@storybook/react';
+import { within, userEvent, screen, expect } from '@storybook/test';
+
+import { Hook } from './stories-helpers';
+import {
+	CreateSnackbarFn,
+	CreateSnackbarFnArgs,
+	SnackbarManager
+} from '../../components/utilities/SnackbarManager';
+import { SELECTORS } from '../../testUtils/constants';
+
+const meta = {
+	component: Hook,
+	subcomponents: { SnackbarManager },
+	decorators: (Story): React.JSX.Element => (
+		<SnackbarManager>
+			<Story />
+		</SnackbarManager>
+	)
+} satisfies Meta<CreateSnackbarFn>;
+
+export default meta;
+
+type Story = StoryObj<CreateSnackbarFnArgs>;
+
+export const HookExample = {
```

Comment:
Same reason, the name seems to ogeneric

Reply:
This is the name that appears after the docs entry in storybook. Since there is just this one story, and it is under the useSnackbar section, I thought HookExample could have been clear enough. But if it is not, I can think of a different name.